src/pages/PoolDetails/PoolDetailsHeader.tsx

```diff
@@ -89,7 +112,7 @@ export function PoolDetailsHeader({
             {token0?.symbol} / {token1?.symbol}
           </ThemedText.HeadlineSmall>
         </Row>
-        {!!feeTier && <FeeTier>{feeTier / BIPS_BASE}%</FeeTier>}
+        {!!feeTier && <FeeTier>{feeTier / 10000}%</FeeTier>}
```

Comment:
what's the difference between `10000` and `10_000`?

Reply:
This factor is unrelated to the `BIPS_BASE` field and I didn't want to correlate the two

Reply:
it looks like feeTier *is* specified in bips, so it *would* be appropriate to use `BIPS_BASE`. Is that wrong?

Reply:
Looking back it does seem like this has no impact. I originally thought this was a regression from the fee work as this change occurred there but PDP was separate from adding fees and was just caused by confusion between uni fees and feeTier